### PR TITLE
chore(starters): add gatsby-starter-quiet

### PR DESCRIPTION
**Pull request by zhouyuexie**
## Starter

Add new starter `gatsby-starter-quiet`

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -3033,3 +3033,20 @@
     - Includes plugins for offline support out of the box
     - TypeScript
     - Prettier & eslint to format & check the code
+- url: https://quietboy.net
+  repo: https://github.com/zhouyuexie/gatsby-starter-quiet
+  description: Gatsby out of the box blog, use TypeScript and highly customized style.
+  tags:
```
Comment (wardpeet) [inline docs/starters.yml]: we don't support all these tags. What do you think of the following?

```json
tags:
  - Typescript
  - Styling:SCSS
  - Linting
```

for a full list you can checkout https://www.gatsbyjs.org/starters/

File: docs/starters.yml [hunk 2]
```diff
@@ -3033,3 +3033,20 @@
     - Includes plugins for offline support out of the box
     - TypeScript
     - Prettier & eslint to format & check the code
+- url: https://quietboy.net
```
[the remark this answers is not in the document]
Comment (zhouyuexie) [reply docs/starters.yml]: Very strange, I can open it here. . .
Do you have a response to ping this domain name?

File: docs/starters.yml (hunk 1 above)
Comment (zhouyuexie) [reply docs/starters.yml]: I am going to modify it.

**Comment by zhouyuexie** [comment]
Since one of the goals of this template is out of the box, most styles are
covered by adding styled-component and sass in consideration of the scenes
that the user may use.

开 2019年6月18日 在 15:03:29, Ward Peeters (notifications@github.com) 写到:

I also have a question about styled-components. What's the reason for
adding styled-components instead of using SASS? You're using 99% sass and
than for one component switched to styled-components. In my eyes it doesn't
seem a good choice.

**Pull request merged**